zebedee-cms/src/main/java/com/github/onsdigital/zebedee/model/approval/ApproveTask.java

```diff
@@ -541,4 +545,38 @@ protected String findCorrectDatasetVersion(List<String> listOfUris) {
         }
         return datasetVersion;
     }
+
+    protected String removeEncoding(String filename) {
```

Comment:
Could "filename" be changed to lower camel case please e.g. "fileName"

Reply:
This is now addressed.